site/content/entry/yuzu-mini-dev-2/index.md

```diff
@@ -0,0 +1,133 @@
+We have to write very annoying workarounds to get that functionality working in the emulator.
+
+<p style="color:cyan"><b><i>Here are some sneak peeks from my Vulkan test build. It's still missing a lot of things, but it's not too bad either.</b></i></p>
+
+{{< imgs
+   "./botw.png|The Legend of Zelda - Breath of the Wild"
+   "./onepiece.png|One Piece - Unlimited World"
+   "./smo.png|Super Mario Odyssey"
+>}}
+
+**Q: What motivates you every day to keep working on this stuff?**
+
+**R:** Emulating modern GPU features on hardware with those same features, is something that has only been possible in the recent times, due to the increasing similarities in hardware between consoles and desktops.
+Slowly understanding how some known functionality is implemented by the hardware, and then re-implementing it on the emulator afterwards, really motivates me.
+
+There are two things that I enjoy the most:
+- seeing games run faster and better than how they ran before and
+- seeing games get more and more closer to how they look on the console.
+
+And if I manage to get these improvements by writing better code, it's an added bonus.
+
+**Q: Can you tell us a little about what you are currently working on? A sneak-peek perhaps?**
+
+**R:** That's classified information :P
+I've recently worked on a improved & generic texture cache. 
+Blinkhawk helped by cleaning up my bugs to get the thing working and he's done an awesome job.
+A generic texture cache allows us to share the complex part of it across graphics APIs.
+That means that we will have the same overall logic for OpenGL and Vulkan, making it easier to maintain and improve.
+
+It's not as good as Dolphin's VideoCommon, one of the better examples for generic graphics code, but it's still better than having to copy-paste code everywhere.
+This is one of the last pre-requirements for the Vulkan API.
+
+I've also been working on implementing various compute shaders related instructions like shared memory (ARB_compute_shader), atomic operations, memory shuffles (NV_shader_thread_shuffle), votes (NV_gpu_shader5, NV_shader_thread_group), surface operations (SUATOM, SULD).
+These are all features from the D3D11/12 era, meaning that we are reaching interesting complexity on the shaders we find in recently released games.
+I am also investigating how to implement some missing functionality in yuzu, but that's a story for another time.
+
+**Q: Would you like to say something to our audience?**
+
+**R:** Well, there are three things I'd like to say.
+
+`First and foremost...` most of our GPU work has been possible thanks to research done by the people working on nouveau (mesa's Nvidia free driver) and libnx's nouveau port to Switch. They have been of huge help.
+
+`Second...` many people believe that Vulkan will bring lots of performance improvements to the emulator, I was one of those too. Don't get me wrong, it might bring some boost on some hardware vendors. 
```

Comment:
Change to `improvements to the emulator... I was one of those too.`

Change next line to:

`Don't get me wrong, it might bring some boost on some hardware vendors, but the main issue resides in the Vulkan's design.`